tensorflow/contrib/go/tensorflow.swigcxx

```diff
@@ -33,8 +32,6 @@ limitations under the License.
   $2 = $input.len;
 }
 
-%include "tensorflow/core/public/version.h"
-%include "tensorflow/core/public/tensor_c_api.h"
+%include "../../c/c_api.h"
```

Comment:
Relative includes are not used in our source code. Mind making it an absolute include?